apps/sms/views/shared/js/localization_helper.js

```diff
@@ -3,12 +3,16 @@
   'use strict';
 
   /**
-   * Look for any iframes and localize them - mozL10n doesn't do this
+   * Look for attachment iframes and localize them - mozL10n doesn't do this
    * XXX: remove once bug 1020130 is fixed
    */
-  function localizeIFrames() {
-    Array.forEach(document.querySelectorAll('iframe'), function(iframe) {
-      var doc = iframe.contentDocument;
+  function localizeAttachmentIFrames() {
```

Comment:
This change is to avoid localizing our hidden iframe, if I'm not mistaken we use this code only for attachment containers?